addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16083,13 +16093,13 @@ msgstr ""
 #. Description of setting "Videos -> Playback -> Playback mode of stereoscopic videos" with label #36520
 #: system/settings/settings.xml
 msgctxt "#36537"
-msgid "Select in which mode stereoscopic 3D videos will be played. [Ask me] will show a dialog to select the desired mode for each playback. [Preferred mode] will use of the preferred mode specified in the 'System -> Video Hardware' section of the settings. [Monoscopic / 2D] will play the video in mono/2D. [Ignore] disables any stereoscopic 3D processing and handling"
+msgid "Select in which mode stereoscopic 3D videos will be played. [Ask me] Will show a dialogue to select the desired mode for each playback. [Preferred mode] Will use of the preferred mode specified in the \"System -> Video hardware\" section of the settings. [Monoscopic / 2D] Will play the video in mono/2D. [Ignore] Disables any stereoscopic 3D processing and handling"
```

Comment:
they might still wrap - depending on the width of the textbox in the skin. But fine with me.